versions/draft-3/SPEC.md

```diff
@@ -1515,6 +1522,130 @@ workflow wf {
 
 In this example, the fully-qualified names that would be exposed as workflow outputs would be `wf.task1.results`, `wf.altname.value`.
 
+## Struct Definition
+A struct is a C-like construct which enables the user to create new compound types that consist of previously existing types. Structs
+can then be used within a `Task` or `Workflow` definition as a declaration in place of any other normal types. The struct takes the place of the
+`Object` type in many circumstances and enables proper typing of its members.
+
+Structs are defined using the `struct` keyword and have a single section consiting of typed declarationns. They are evaluated first prior
+to workflows and tasks.
```

Comment:
I think it's implied here, but I think we should be explicit (especially since @yfarjoun already wrote one inline in one of the examples...):

> "Structs are declared separately, outside of any `workflow` or `task` definition". They are part of the namespace of the WDL file they are written in.

Reply:
Good catch. I think making this explicit is quite important